slang-gfx.h

```diff
@@ -46,8 +47,8 @@ typedef SlangResult Result;
 typedef SlangInt Int;
```

Comment:
Do we still use these? If not we can probably just get rid of it?

Reply:
Unsure, but no real harm in leaving it in for now, I think